pyproject.toml

```diff
@@ -55,6 +55,7 @@ sphinx-copybutton = { version = "^0.5.0", optional = true }
 sphinx-intl = { version = "^2.1.0", optional = true }
 myst-nb = { version = "^0.17.1", optional = true }
 furo = { version = ">=2022,<2024", optional = true }
+pillow = "^10.0.0"
```

Comment:
This is a nitpick, it would be better to move pillow up with the other non-optional dependencies.

Reply:
Moved upwards, thanks for the idea.